app/src/main/java/com/epfl/drawyourpath/mainpage/fragments/DrawFragment.kt

```diff
@@ -1,8 +1,36 @@
 package com.epfl.drawyourpath.mainpage.fragments
 
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import com.epfl.drawyourpath.R
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
 
-class DrawFragment : Fragment(R.layout.fragment_draw) {
+class DrawFragment : Fragment(R.layout.fragment_draw), OnMapReadyCallback {
     // TODO: add Google Map UI here and the ViewModel for this screen/fragment.
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val mapFragment = childFragmentManager
+            .findFragmentById(R.id.fragment_draw_map) as SupportMapFragment
+        mapFragment.getMapAsync(this)
+    }
+
+    override fun onMapReady(p0: GoogleMap) {
+        val coordinates = LatLng(46.5185, 6.56177)
+        p0.addMarker(
+            MarkerOptions()
+                .position(coordinates)
+                .title("EPFL")
+        )
```

Comment:
(optional) It would be good to create constants for the numbers, although if I understand correctly, the code would be removed in the future as it's just an example.

Reply:
This is just an example to show that the map works and that we can use it in code. So yes it will be removed in the next issue.